packages/preview/codedis/0.1.0/lib.typ

```diff
@@ -0,0 +1,69 @@
+#let code(
+  code, 
+  lang: "py",
+  stroke: luma(170), // Stroke color
+  fill_1: luma(250), // First line block fill
+  fill_2: luma(240), // Second line block fill
```

Comment:
It is recommended to use kebab-case names for public functions and their arguments, but it's ultimately up to you.
```suggestion
  fill-1: luma(250), // First line block fill
  fill-2: luma(240), // Second line block fill
```

Reply:
`fill1` and `fill2` would work as well.